.gitmodules

```
@@ -12,7 +12,9 @@
 [submodule "core/packages/contracts/lib/openzeppelin-contracts"]
 	path = core/packages/contracts/lib/openzeppelin-contracts
 	url = https://github.com/openzeppelin/openzeppelin-contracts
-	branch = v4.8.2
 [submodule "core/packages/contracts/lib/canonical-weth"]
 	path = core/packages/contracts/lib/canonical-weth
 	url = https://github.com/snowfork/canonical-weth
+[submodule "lib/openzeppelin-contracts"]
+	path = lib/openzeppelin-contracts
```

Should this be in `core/packages/contracts`?
```suggestion
	path =  core/packages/contracts/lib/openzeppelin-contracts
```